profiles/features/musl/package.mask

```diff
@@ -65,3 +65,7 @@ www-client/firefox-bin
 # Mikle Kolyada <zlogene@gentoo.org> (2020-03-20)
 #  No source builds for musl
 app-emulation/firecracker
+
+# Hank Leininger <hlein@korelogic.com> (2021-07-14)
```

Comment:
Nit: shift to the top of the file (below the comment block, so above the most-recent entry)

Reply:
Thanks! I could not tell because existing entries seemed haphazardly ordered, and my google-fu failed for finding package.mask update guidance (which doesn't come up often, after all). Noted for the future.

Reply:
There's this [page](https://devmanual.gentoo.org/profiles/package.mask/index.html) which of course doesn't mention it yet (will soon)!

Reply:
https://github.com/gentoo/devmanual/pull/236